src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -41,6 +43,8 @@
     public static final String CANNOT_CONNECT_TO_INSTANCE = "Cannot connect to instance ";
     private static final String AD_CONFIG_SEP = "#### AUTO-DISCOVERY ####";
     private static final String AD_LEGACY_CONFIG_SEP = "#### SERVICE-DISCOVERY ####";
+    private static final String AD_CONFIG_TERM = "#### AUTO-DISCOVERY TERM ####";
+    private static final String AD_LEGACY_CONFIG_TERM = "#### SERVICE-DISCOVERY TERM ####";
```

Comment:
The end header is `"#### SERVICE-DISCOVERY END ####\n"` and not `#### SERVICE-DISCOVERY TERM ####`.
from: https://github.com/DataDog/dd-agent/pull/3426/files#diff-05b1a4892667b4d4a36c7ac84208c8c0R77